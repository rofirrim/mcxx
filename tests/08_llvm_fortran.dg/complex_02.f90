! <testinfo>
! test_generator="config/mercurium-fortran-llvm run"
! test_stdout="complex_02.f90.out"
! </testinfo>
PROGRAM MAIN
  IMPLICIT NONE
  COMPLEX :: C, D, E

  C = (2, 3)
  D = (4, 5)

  E = C ** D
  PRINT *, E

  E = D ** C
  PRINT *, E
END PROGRAM MAIN
