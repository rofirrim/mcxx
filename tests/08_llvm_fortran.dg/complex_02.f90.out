 (-0.753046215    ,-0.986428261    )
 (  1.31593108    ,  2.45815659    )
